package.json

```diff
@@ -99,7 +99,7 @@
     "@hot-loader/react-dom": "^16.13",
     "@reach/auto-id": "^0.13.0",
     "@reach/combobox": "^0.12.1",
-    "@reach/menu-button": "0.7.4",
+    "@reach/menu-button": "0.8.6",
     "@reach/rect": "^0.16.0",
     "@reach/tabs": "^0.1.5",
```

Comment:
Would be nice to be able to see what I'm paying for without having to first click on it (i.e. price tag blocking the sticker)

![image](https://user-images.githubusercontent.com/64950861/150264035-147a97a6-8962-46bb-b666-000039ae29c8.png)

Reply:
All the stickers were messed up because of a class conflict, not just the paid ones. Thanks!

![image](https://user-images.githubusercontent.com/94839278/150337423-cabec56b-bba5-4a55-877f-d61e035202d5.png)